docs/release-notes/2.1.0.asciidoc

```diff
@@ -33,6 +33,8 @@
 * Do not attempt rolling upgrades for non-HA Elasticsearch clusters {pull}5327[#5327] (issue: {issue}5321[#5321])
 * Use precondition when deleting secrets {pull}5273[#5273] (issue: {issue}5249[#5249])
 * Support new Agent base image as of 7.17 {pull}5268[#5268]
+* Fix Webhooks match policy within Openshift {pull}5437[#5437] (issue: {issue}5423[#5423])
```

Comment:
I would adjust the name to something like "Fix webhook match policy for OLM based installations" or something like that. This is because the issue is not OpenShift specific. All OLM based installations were affected. (admittedly the vast majority of those will be on OpenShift)

Reply:
done.